appveyor/mozillaSyms.py

```diff
@@ -0,0 +1,102 @@
+# Based on code from https://gist.github.com/luser/2ad32d290f224782fcfc
+
+"""Script to convert and upload appropriate NVDA debug symbols to Mozilla crash-stats.
+This should just be run as a script with no arguments.
+It expects the crash-stats auth token to be placed in the mozillaSymsAuthToken environment variable.
+To update the list of symbols uploaded to Mozilla, see the DLL_NAMES constant below.
+"""

```

Comment:
Normally, for sure. However, the original code didn't have a copyright or license statement, so I'm not sure what to put here. I figured it was best just to attribute it. I don't think we really care what people do with this particular piece of code, so I'm not too concerned.